### PR TITLE
Add update to run on linux

### PR DESCRIPTION
**Pull request by KojoBoat**

### CODE REVIEW

**Comment by bingenito** [comment]
Hi @KojoBoat, thanks for the PR!  Can you please follow the steps in the contributing guide (https://github.com/morganstanley/Crossroads/blob/main/CONTRIBUTING.md) and add a DCO file and reference it with your commit.  Thank you.

**Comment by KojoBoat** [comment]
Hello Brain,

Thanks for the response. I will do just that. Thank you.

Best Regards,
Christian.

**Comment by KojoBoat** [comment]
Brian**  sorry about the typo :)

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ on:
 jobs:
   build:
 
-    runs-on: windows-latest
+    runs-on: ['windows-latest', 'ubuntu-latest']
```
Comment (bagdxk) [inline .github/workflows/continuous-integration.yml:14]: matrix is required for multiple OS.

**Comment by bingenito** [comment]
Closed and replaced with #12

**Pull request closed without merging**